### ISSUE TITLE
Flatpak so Mercury is available on all Linux OS'

### ISSUE THREAD

**Issue by slrgt** [Alex313031/Mercury#165]
Hi, the Mercury project is extremely interesting and I would love to use it on Bazzite (a fedora fork) but I noticed theres no rpm package or a flatpak. Could a Flatpak be made available so we can use it on any Linux OS?

**Comment by BrianAllred** [comment]
I agree a Flatpak would be great, but in the meantime, you can use it on Bazzite via Distrobox pretty easily, that's what I did when I used Bazzite. Create an Arch Linux Distrobox and install Mercury from the AUR, works great.

**Comment by superuser-miguel** [comment]
I'm on Fedora, and was looking for an RPM, so basically would need to create a Debian image then install it and all it's dependencies. . .

**Comment by Mondrethos** [comment]
Also on Bazzite, which is immutable. A Flatpak would make it much less complicated to run.

**Comment by Novantric** [comment]
Yeah please. No reason not to considering other forks are on flathub too.
